### ISSUE TITLE
[CAIA A11y Research - PILOT]: Ask VA - Screen Magnification Pilot with Figma Prototype (End of April 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#81692]
> [!IMPORTANT]
> - **TICKET STATUS:** `SCHEDULED`
> - **PILOT DATE:**  `4/30/2024`
> - **TYPES OF ASSISTIVE TECH:** `FIGMA PROTOTYPE`

# CAIA A11y Support
This is a child ticket of the research ticket [80535](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80535) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)
@coforma-jamie 

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Ask VA VA.gov
- OCTO-DE product owner: Becky Phung
- Product manager: Holly Hansen
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Hemesh Patel, Eddie Otero
- Product/team Slack channel: #ask-va-public
- **Dedicated content writer on your team (if you have one):** `writer`
- **Dedicated a11y specialist on your team (if you have one):** `a11y`

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** `4/30/2024 @ 10 A.M.`
- **Prototype**: `FIGMA` [Prototype – WIP](https://www.figma.com/file/SYvmqLiAKaGtIcePSRRimB/Ask-VA-Prototype-R2?type=design&mode=design&t=hk9eNPHEIgYj7Jy9-0)
- **Research Convo Guide**: [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/04-2024%20Progress%20bar/Conversation%20guide.md)
- **Research Plan**: [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/04-2024%20Progress%20bar/Research%20plan.md)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** Jamie
- [x] **Determine date and time for pilot:** 4/30/2024
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review prototype** in `Figma` - `WIP`
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/04-2024%20Progress%20bar/Conversation%20guide.md) and provide feedback in comments.
- [x] Request AT device and tech to be used in sessions
- [x] Receive AT device and tech from Team in research ticket to be used in sessions to prep for pilot
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. Folder for Team in GH Deliverables for CAIA A11Ys.
```

**Comment by coforma-jamie** [comment]
No comments on the convo guide, it looks great. If anything comes up during our session, I'll add comments here :)

**Comment by coforma-jamie** [comment]
Great session @tygindraux!

Main things to keep in mind are:
- There are different ways to magnify: "blow up" the text in-browser, "blow up" interface elements generally across the desktop, use a "magnifying glass". I used a "magnifying glass" for this session.
- When you're zoomed in, you're only seeing a chunk of content at a time. And if you're using a "magnifying glass," you aren't seeing the interface unless that's where your cursor happens to be - you might not know that there's a scrollbar, for example.
- The tool I used kept me 'focused' on where my cursor was, NOT the top of the screen, on page load. Others may work differently. But I'd look out for that - do folks know that they're logged in (if they don't see the page header), or that they need to hit a submit button to complete the form?

**Comment by tygindraux** [comment]
Thank you Jamie! Great to get your feedback – and to learn more about how screen magnification users might experience the product.
